### PR TITLE
style(API nav): Improve doc/section titles

### PR DESCRIPTION
**Pull request by austin-schaefer**
Simplifies and clarifies navigation titles in the NerdGraph category, which are currently a bit redundant and sometimes wrong

### CODE REVIEW

**Review by nbaenam — DISMISSED** [review]
It looks good to me, although shouldn't we try to be consistent with the left nav title and the page title? These pages don't follow this convention, for example:
 - title: Policies > NerdGraph tutorial: alert policies
-  title: Cloud integrations tutorial > NerdGraph tutorial: Configure cloud integrations

**Comment by austin-schaefer** [comment]
> It looks good to me, although shouldn't we try to be consistent with the left nav title and the page title? These pages don't follow this convention, for example:
> 
>     * title: Policies > NerdGraph tutorial: alert policies
> 
>     * title: Cloud integrations tutorial > NerdGraph tutorial: Configure cloud integrations

Good call @nbaenam , just pushed up a commit to preface all titles with `NerdGraph tutorial`

**Review by nbaenam — DISMISSED** [review]

**Review by nbaenam — APPROVED** [review]
@austin-schaefer, I fixed a couple of mdx errors with a commit. Hope you don't mind.

**Comment by austin-schaefer** [comment]
Nice catch, thank you @nbaenam !

**Comment by austin-schaefer** [comment]
@nbaenam : I decided to spend a bit more time and see what it would look like to remove the "Other tutorials" section of NG and just organize it alphabetically, I'd love your thoughts! You can see the new category here:

https://deploy-preview-17486--docs-website-netlify.netlify.app/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/

File: src/nav/telemetry-data-platform.yml [hunk 1]
```diff
@@ -137,64 +167,34 @@ pages:
...
-              - title: Alert notification channels
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-alert-notification-channels
-              - title: Alerting loss of signal
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling
-              - title: Nerdgraph tutorial
-                path: /docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github
-              - title: Incident correlation topology
-                path: /docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial
-          - title: 'NerdGraph: logs'
+              - title: View trace details
```
Comment (nbaenam) [inline src/nav/telemetry-data-platform.yml:172]: Nav title: `View trace details`
Doc title: `NerdGraph tutorial: Viewing distributed trace details`
For consistency, I'd write or `view` or `viewing`.

**Review by nbaenam — APPROVED** [review]
Hey @austin-schaefer, I added some suggestions! Feel free to take them or leave them🙂

**Comment by ally-sassman** [comment]
@austin-schaefer Is this ready to be merged in?

**Comment by nbaenam** [comment]
Hey @austin-schaefer, any update on this? I can fix all the conflicts except the navigation one. Can you please check this? Thanks a lot!

File: src/nav/telemetry-data-platform.yml [hunk 2]
```diff
@@ -105,7 +105,23 @@ pages:
...
+              - title: Manage accounts
+                path: /docs/apis/nerdgraph/examples/manage-accounts-nerdgraph
+              - title: Manage users
+                path: /docs/apis/nerdgraph/examples/nerdgraph-manage-users
+              - title: Manage user groups
+                path: /docs/apis/nerdgraph/examples/nerdgraph-manage-groups
+              - title: Manage tags
+                path: /docs/apis/nerdgraph/examples/nerdgraph-tagging-api-tutorial
+              - title: Workloads
+                path: /docs/apis/nerdgraph/tutorials/nerdgraph-workloads-api-tutorials
```
[the remark this answers is not in the document]
Comment (austin-schaefer) [reply src/nav/telemetry-data-platform.yml:123]: Good catch!

File: src/nav/telemetry-data-platform.yml [hunk 3]
```diff
@@ -115,17 +131,31 @@ pages:
...
-              - title: Historical data export queries
-                path: /docs/apis/nerdgraph/examples/nerdgraph-historical-data-export
-          - title: 'NerdGraph: dashboards'
+              - title: 'Channels'
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-notifications-channels
+              - title: 'Destinations'
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-notifications-destinations
+              - title: Incident correlation topology
+                path: /docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial
+              - title: Issues and incidents
```
[the remark this answers is not in the document]
Comment (austin-schaefer) [reply src/nav/telemetry-data-platform.yml:142]: Hmm, so in this case I think the mismatch is actually right. In the longer title it's a pair of noun phrases (`Issue... query API` and `Incident query API`) where a singular feels right. Whereas in the short title the focus is on the things being interacted with, and in that case the singular `Issue and incident` would seem to imply there is only one issue and only one incident. Happy to be wrong here though!

File: src/nav/telemetry-data-platform.yml [hunk 4]
```diff
@@ -115,17 +131,31 @@ pages:
...
-                path: /docs/apis/nerdgraph/examples/async-queries-nrql-tutorial
-              - title: Historical data export queries
-                path: /docs/apis/nerdgraph/examples/nerdgraph-historical-data-export
-          - title: 'NerdGraph: dashboards'
+              - title: 'Channels'
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-notifications-channels
+              - title: 'Destinations'
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-notifications-destinations
+              - title: Incident correlation topology
+                path: /docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial
```
[the remark this answers is not in the document]
Comment (austin-schaefer) [reply src/nav/telemetry-data-platform.yml:141]: Looks like someone fixed this one in the meantime

File: src/nav/telemetry-data-platform.yml [hunk 5]
```diff
@@ -115,17 +131,31 @@ pages:
...
+              - title: Issues and incidents
+                path: /docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github
+              - title: Loss of signal
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling
+              - title: Notification channels
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-alert-notification-channels
+              - title: NRQL alert conditions
```
[the remark this answers is not in the document]
Comment (austin-schaefer) [reply src/nav/telemetry-data-platform.yml:148]: Good catch. Looks like the doc uses `NRQL condition` so I'll use that in the taxonomy.

File: src/nav/telemetry-data-platform.yml [hunk 6]
```diff
@@ -115,17 +131,31 @@ pages:
...
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-nrql-condition-alerts
+              - title: Policies
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-alerts-policies
+              - title: 'Workflows'
+                path: /docs/apis/nerdgraph/examples/nerdgraph-api-workflows
+          - title: Cloud integrations tutorial
+            path: /docs/apis/nerdgraph/examples/nerdgraph-cloud-integrations-api-tutorial
+          - title: Concurrent request limits
+            path: /docs/apis/nerdgraph/examples/nerdgraph-concurrency-tutorial
+          - title: 'Dashboards'
```
[the remark this answers is not in the document]
Comment (austin-schaefer) [reply src/nav/telemetry-data-platform.yml:158]: All good catches, thank you!

File: src/nav/telemetry-data-platform.yml (hunk 1 above)
Comment (austin-schaefer) [reply src/nav/telemetry-data-platform.yml:172]: Nice work!

**Comment by austin-schaefer** [comment]
Thanks for the nudge @nbaenam ! Just fixed the merge conflicts and went back to address your peer edit feedback.

**Review by paperclypse — APPROVED** [review]
Works for me!

**Pull request merged**